### PR TITLE
adding mysql

### PR DESCRIPTION
**Pull request by NoaJinko**

### CODE REVIEW

File: src/main/java/Sample/Main.java [hunk 1]
```diff
@@ -33,13 +33,16 @@ public static void main(String[] args) {
         boys_and_girls.renumber(sexIsNotImportant);
         boys_and_girls.print_personArrayList(sexIsNotImportant);
 
-        System.out.println();
-        System.out.println("Записываем содержимое ArrayList в txt.");
-        boys_and_girls.toTxt("Test.txt");
+//        System.out.println();
```
Comment (NoaJinko) [inline src/main/java/Sample/Main.java]: remove comments

File: src/main/java/Sample/Main.java [hunk 2]
```diff
@@ -33,13 +33,16 @@ public static void main(String[] args) {
...
+//        System.out.println("Записываем содержимое ArrayList в txt.");
+//        boys_and_girls.toTxt("Test.txt");
 
-        System.out.println();
-        System.out.println("Сортируем по возрасту и записываем результат в соответствующие txt.");
-        boys_and_girls.sortingByAge_lessThan20(sexIsNotImportant);
-        boys_and_girls.sortingByAge_moreThan20(sexIsNotImportant);
+//        System.out.println();
```
Comment (NoaJinko) [inline src/main/java/Sample/Main.java]: remove comments

File: src/main/java/Sample/FromExcel.java [hunk 3]
```diff
@@ -96,4 +98,24 @@ public void sortingByAge_moreThan20 (List<Person> persons) {
             ie.printStackTrace();
         }
     }
+
+    public void adding_to_DB (List<Person> persons) {
+        try {
```
Comment (NoaJinko) [inline src/main/java/Sample/FromExcel.java]: maybe should you shoul include DB connect logic in separate method?

Comment (IvanovNd) [inline src/main/java/Sample/FromExcel.java]: Did you forget about my comment? )

**Pull request closed without merging**